spark/src/main/scala/org/apache/spark/sql/delta/commands/MergeIntoCommandBase.scala

```diff
@@ -89,6 +89,40 @@ trait MergeIntoCommandBase extends LeafRunnableCommand
     }
   }
 
+  /**
+   * Output expressions from the target adjusted after applying schema evolution. There are two
```

Comment:
This could be even clearer. "Output expressions from the target" can be interpreted in multiple ways. IIUIC this produces expressions to convert from a pre-evolution target row to the post-evolution target row, for use to copy rows unmodified, as well as for default expressions for columns that are not modified updated rows. Is that right?

Reply:
Correct, that's much better, I rephrased that comment